apps/shelter-web/tsconfig.app.json

```diff
@@ -19,5 +19,11 @@
     "src/**/*.spec.jsx",
     "src/**/*.test.jsx"
   ],
-  "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx"]
+  "include": [
+    "src/**/*.js",
+    "src/**/*.jsx",
+    "src/**/*.ts",
+    "src/**/*.tsx",
+    "types/**/*.d.ts"
```

Comment:
Think we can revert the **/*.d.ts in app tsconfigs now given we don't need to import the custom one anymore.